tests/test_otio_openassetio.py

```diff
@@ -115,29 +127,20 @@ def bal_linker_args():
     }
 
 
-@pytest.fixture(autouse=True)
-def bal_plugin_env(base_dir, monkeypatch):
-    """
-    Provides a modified environment with the BasicAssetLibrary
-    plugin on the OpenAssetIO search path based on the expected
-    dependencies install location
-    """
-    plugin_dir = os.path.join(
-        base_dir,
-        "dependencies",
-        "OpenAssetIO",
-        "resources",
-        "examples",
-        "manager",
-        "BasicAssetLibrary",
-        "plugin",
-    )
-    monkeypatch.setenv("OPENASSETIO_PLUGIN_PATH", plugin_dir)
-
-
 @pytest.fixture()
-def base_dir():
+def bal_linker_args_missing_asset():
```

Comment:
[consider] Might be simpler to have alternate raw OTIO string(s) with an invalid ref/missing ref, vs needing the extra libraries/fixtures?

Reply:
I'm not so sure on this myself, the alternate raw strings seem messier to my eye.
If it's all the same to you, (and you agree it's testing the same code paths,) i'd rather keep the segregated data files as the source of variation